app/build.gradle

```diff
@@ -70,6 +70,8 @@ dependencies {
     // Hilt
     implementation "com.google.dagger:hilt-android:$hilt_version"
     kapt "com.google.dagger:hilt-compiler:$hilt_version"
+    kapt "androidx.hilt:hilt-compiler:1.0.0"
+    implementation 'androidx.hilt:hilt-navigation-compose:1.0.0'
```

Comment:
Can we use `hilt_version` rather than the hardcoded values here?

Reply:
Copy that, will update the dependencies to use `hilt_version` in the next push

Reply:
I created a variable `androidx_hilt_version` to use instead of hardcoded values. I have to use "1.0.0" of the androidx.hilt:hilt-compiler, see here: https://androidx.tech/artifacts/hilt/hilt-compiler/ . So that is why I created a new variable here.